docs/builtins.md

```diff
@@ -1415,20 +1419,21 @@ Output
     color c:string
 
 The `color` function changes the color of the _stroke_ and the _fill_ to
-the given CSS color string `c`. Evy supports all
-[CSS color values](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value),
-including semi-transparent ones. For example, the following code changes
-the color to a shade of red that is 60% opaque:
-`color "hsl(0deg 100% 50% / 60%)"`.
+the given CSS color string `c`. Evy supports all[CSS color values],
+including semi-transparent ones. For example, the following code
+changes the color to a shade of red that is 60% opaque: 
+`color "hsl (0deg 100% 50% / 60%)"`.
 
 _Named CSS colors_, such as `"red"`, `"darkmagenta"`, and
-`"springgreen"`, are a simpler way of specifying
-common colors. For a complete list of named CSS colors, see the
-[Mozilla Developer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color).
+ `"springgreen"`, are a simpler way of specifying common colors. For a
```

Comment:
how come this line and the following ones became indented 1 char?

Reply:
no idea. i think that's my editors line indentation... fixed.